target/linux/ath79/image/generic.mk

```diff
@@ -694,6 +694,19 @@ define Device/librerouter_librerouter-v1
 endef
 TARGET_DEVICES += librerouter_librerouter-v1
 
+define Device/mikrotik_routerboard-wap-g-5hact2hnd
+  $(Device/mikrotik)
+  SOC := qca9556
+  DEVICE_MODEL := RouterBOARD wAP G-5HacT2HnD (wAP AC)
+  IMAGE_SIZE := 16000k
```

Comment:
@rogerpueyo 0xfe0000 != 16000k . Can I update it to 16256k?

Reply:
?

Reply:
Yes, the firmware partition is actually 0xfe0000 sized, same as in ar71xx. This 16000k value came directly from the ar71xx support, it could be that some nor-spi device there was sized like that but we can definitely set 16256k here, please.

Reply:
Okay, pushed that to my staging branch.